server/endpoints_v1.go

```diff
@@ -178,15 +178,15 @@ func (server *HTTPServer) addV1RuleEndpointsToRouter(router *mux.Router, apiPref
 	router.HandleFunc(apiPrefix+DisableRuleForClusterEndpoint, server.proxyTo(
 		aggregatorBaseEndpoint,
 		&ProxyOptions{RequestModifiers: []RequestModifier{
-			server.extractUserIDOrgIDFromTokenToURLRequestModifier(ira_server.DisableRuleForClusterEndpoint),
+			server.extractOrgIDFromTokenToURLRequestModifier(ira_server.DisableRuleForClusterEndpoint),
 			checkRuleIDAndErrorKeyAreValid(),
 		}},
 	)).Methods(http.MethodPut, http.MethodOptions)
 
 	router.HandleFunc(apiPrefix+EnableRuleForClusterEndpoint, server.proxyTo(
 		aggregatorBaseEndpoint,
 		&ProxyOptions{RequestModifiers: []RequestModifier{
-			server.extractUserIDOrgIDFromTokenToURLRequestModifier(ira_server.EnableRuleForClusterEndpoint),
+			server.extractOrgIDFromTokenToURLRequestModifier(ira_server.EnableRuleForClusterEndpoint),
```

Comment:
sorry I don't know the contex for this: extractUserIDOrgIDFromTokenToURLRequestModifier is used in several other places, why it was replaced only here?

Reply:
because the other endpoints are not ready yet. only this one got the user_id removed in https://github.com/RedHatInsights/insights-results-aggregator/pull/1676

Reply:
the `extractUserIDOrgIDFromTokenToURLRequestModifier` will be removed completely after all endpoints stop using it